dashboard/bin/scheduled_ops_emails

```diff
@@ -6,24 +6,38 @@ require_relative '../config/environment'
 def main
   Dir.chdir(dashboard_dir) do
     # Get all workshops that start two weeks or 3 days from today
-    workshops = DASHBOARD_DB.fetch('
+    upcoming_workshops = DASHBOARD_DB.fetch('
       SELECT * FROM segments JOIN workshops ON workshops.id = workshop_id  GROUP BY workshop_id
       HAVING (DATE(MIN(start)) = (DATE_ADD(CURDATE(), INTERVAL 2 WEEK)))
       OR (DATE(MIN(start)) = (DATE_ADD(CURDATE(), INTERVAL 3 DAY)));
     ')
-    workshops.each do |workshop|
-      workshop_segments = DASHBOARD_DB.fetch("SELECT * FROM segments WHERE workshop_id = #{workshop[:id]};")
-      teachers = Workshop.find(workshop[:id]).teachers
-      drop_ins = Workshop.find(workshop[:id]).unexpected_teachers
-      facilitators = Workshop.find(workshop[:id]).facilitators
-      [teachers, drop_ins, facilitators].each do |recipient_list|
-        recipient_list.each do |recipient|
-          workshop[:segments] = workshop_segments
-          puts("Sending email to #{recipient.properties['ops_first_name']}")
-          OpsMailer.workshop_reminder(workshop, recipient).deliver
+    ending_workshops = DASHBOARD_DB.fetch('
+      SELECT * FROM segments JOIN workshops ON workshops.id = workshop_id  GROUP BY workshop_id
+      HAVING (DATE(MAX(start)) = CURDATE());
+    ')
+
+    [upcoming_workshops, ending_workshops].each do |workshop_list|
+      workshop_list.each do |workshop|
+        workshop_segments = DASHBOARD_DB.fetch("SELECT * FROM segments WHERE workshop_id = #{workshop[:id]};")
+        teachers = Workshop.find(workshop[:id]).teachers
+        drop_ins = Workshop.find(workshop[:id]).unexpected_teachers
+        facilitators = Workshop.find(workshop[:id]).facilitators
+        [teachers, drop_ins, facilitators].each do |recipient_list|
+          recipient_list.each do |recipient|
+            workshop[:segments] = workshop_segments
```

Comment:
do you use :segments here? I don't see any in exit_survey_information, do you only use it in workshop_reminder?